show/platform.py

```diff
@@ -104,7 +104,9 @@ def psustatus(index, json, verbose):
 def ssdhealth(device, verbose, vendor):
     """Show SSD Health information"""
     if not device:
-        device = os.popen("lsblk -o NAME,TYPE -p | grep disk").readline().strip().split()[0]
+        _, stdout = getstatusoutput_noshell_pipe(["lsblk", "-o", "NAME,TYPE", "-p"], ["grep", "disk"])
+        stdout_first_line = stdout.split('\n')[0]
```

Comment:
>[0] [](http://example.com/codeflow?start=46&length=3)

If stdout is empty, index 0 will be out of range, so there is risk of exception?